cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -2043,6 +1988,12 @@ CreateContainerCmd createContainer(
             DockerClient client)
         {
             ExposedPort exposedPort = ExposedPort.tcp(9092);
+            String volume = "/bitnami/kafka";
+
+            client.createVolumeCmd()
+                .withName(ZILLABASE_KAFKA_VOLUME)
+                .withLabels(Map.of("io.aklivity", "zillabase"))
```

Comment:
```suggestion
                .withLabels(Map.of("io.aklivity.zillabase.cli.project", "my-project"))
```
where the value of `my-project` comes from the directory where zillabase is locating the `zillabase/config.yaml` for example.